compiler/commands.nim

```diff
@@ -901,6 +901,8 @@ proc processSwitch*(switch, arg: string, pass: TCmdLinePass, info: TLineInfo;
     conf.globalOptions.incl optSourcemap
     conf.options.incl optLineDir
     # processOnOffSwitchG(conf, {optSourcemap, opt}, arg, pass, info)
+  of "nimscript2":
```

Comment:
I don't understand this commit 100%.. but looks like --nimconfig2 is the "new nimscript". If so, should this be "nimconfig2" instead of "nimscript2"? Or is "nimscript2" different than "nimconfig2"? :)

Reply:
It's all WIP, don't think about it too hard.